if !(2 > 0)
    throw 'unexpected result';
